### DISCUSSION TITLE
How will the final version work in regards to unlocking/progression?

### DISCUSSION THREAD

**Discussion by MIC132** [open-duelyst/duelyst#249] [Q&A] [answered]
So as it stands, on the staging environment you just get all the cards. I like this very much, as I never was one for grinding to unlock content.

But I assume once prod rolls around the progression system will work as normal (minus any potential paid elements?).
Will there possible be an option to still play in the "just have everything" mode?

**Comment by willroberts** [comment, marked as answer]
I have a WIP branch to move the full collection option behind a config/feature flag, so people can enable/disable it as desired. For the production environment I think it's fine to leave it enabled.

**Reply by MIC132** [reply]
I'm mostly interested how it'll be on the actual server available from your client (after it moves from staging), since I'm not interested in hosting my own for now.
So if I'm reading that correctly, you plan on leaving it at "full collection"?

**Reply by MIC132** [reply]
On that topic, will the data be transferred from the current version? I want to know whether to backup decks, and if I'll lose challenges progress, etc. (I don't care about rank and such though).

**Reply by willroberts** [reply]
There probably won't be any data transfer between staging and production, but you have the option of exporting and importing decks from the collection page.

**Reply by MIC132** [reply]
> There probably won't be any data transfer between staging and production, but you have the option of exporting and importing decks from the collection page.

Will there be some warning before you switch the clients to point to prod?
I guess I'll just keep backups constantly.
I'm mostly gonna be pissed about puzzle progress XD

**Reply by CactiChameleon9** [reply]
Just to chip in, someone who has never played before, as myself, could find it confusing or complicated to have ALL of the cards to begin with. Maybe some sort of client option in settings to unlock all would be better to allow for more normal progression.
